### PR TITLE
Add missing test to member to cover fetching missing coordinates

### PR DESCRIPTION
**Pull request by mike1729**

### CODE REVIEW

File: src/member.rs [hunk 1]
```diff
@@ -839,3 +839,113 @@ where
         let _ = network_exit.send(());
     }
 }
+
+#[cfg(test)]
```
Comment (timorl) [inline src/member.rs:843]: General comment/question, not directly related to the review: do we want to keep tests in the same files as the code? I see that this is nice in some ways (information is nicely concentrated in one place), but also any changes to tests mean the non-test parts of code also have to be recompiled (at least I think so, is that true?). This is not bothersome right now, but will probably be when the codebase and compilation times grow.

File: src/signed.rs [hunk 2]
```diff
@@ -55,7 +55,7 @@ pub trait Signable {
 /// `[PartiallyMultisigned<'a, T, MK>]`.
 #[derive(Clone, Debug, Decode, Encode)]
 pub struct UncheckedSigned<T: Signable, S> {
-    signable: T,
+    pub(crate) signable: T,
```
Comment (timorl) [inline src/signed.rs:58]: I think Misza's changes add a getter for this, which is nicer than making the field (crate)public.

File: src/testing/mock.rs [hunk 3]
```diff
@@ -201,7 +201,7 @@ impl Spawner {
     }
 }
 
-type NetworkData = NetworkDataT<Hasher64, Data, Signature>;
+pub(crate) type NetworkData = crate::NetworkData<Hasher64, Data, Signature>;
```
Comment (timorl) [inline src/testing/mock.rs:204]: Oh, nice, `testing::NetworkData`, <3.

File: src/units.rs [hunk 4]
```diff
@@ -9,7 +9,7 @@ use std::{cell::RefCell, collections::HashMap, hash::Hash as StdHash};
 
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Encode, Decode, StdHash)]
 pub(crate) struct UnitCoord {
-    round: u16,
+    pub(crate) round: u16,
```
Comment (timorl) [inline src/units.rs:12]: I would strongly prefer accessors to making fields public, especially only for tests. Same for the other instances in this file.

**Review by timorl — CHANGES_REQUESTED** [review]
Lovely, the only issue I have is making some fields public – I'd prefer methods for that.

File: src/member.rs (hunk 1 above)
Comment (mike1729) [reply src/member.rs:843]: This file is already quite big and this test is more of an integration test so I'll move it.

File: src/units.rs (hunk 4 above)
Comment (mike1729) [reply src/units.rs:12]: Good point, I'll change that.

File: src/signed.rs [hunk 5]
```diff
@@ -59,6 +59,13 @@ pub struct UncheckedSigned<T: Signable, S> {
     signature: S,
 }
 
+impl<T: Signable, S: Signature> UncheckedSigned<T, S> {
+    #[cfg(test)]
```
Comment (timorl) [inline src/signed.rs:63]: I think you can slap this on even one line higher.

**Review by timorl — APPROVED** [review]

File: src/signed.rs (hunk 5 above)
Comment (mike1729) [reply src/signed.rs:63]: Nope, fmt doesn't allow that.

**Review by DamianStraszak — APPROVED** [review]

**Pull request merged**